packages/docs/src/components/Fiddle/SharedPlayer.ts

```diff
@@ -120,11 +119,14 @@ export async function borrowPlayer(
       ValueDispatcher,
       DefaultPlugin,
     } = await import(/* webpackIgnore: true */ '@motion-canvas/core');
-    const {makeScene2D, LezerHighlighter} = await import(
+    const {makeScene2D, Code, LezerHighlighter} = await import(
       /* webpackIgnore: true */ '@motion-canvas/2d'
     );
-    LezerHighlighter.registerParser(javascript);
-    LezerHighlighter.registerParser(rust, 'rs');
+    Code.defaultHighlighter = new LezerHighlighter(
```

Comment:
I still think we should include this somewhere in the docs, too.

Reply:
Done